src/Registration/IntensityProfileRegistrationForceField.h

```diff
@@ -133,9 +133,6 @@ class IntensityProfileRegistrationForceField : public core::behavior::ForceField
 
     core::behavior::MechanicalState<DataTypes>* getObject() { return this->mstate; }
 
-    static std::string templateName(const IntensityProfileRegistrationForceField<DataTypes,ImageTypes>* = NULL) { return DataTypes::Name()+ std::string(",")+ImageTypes::Name();    }
```

Comment:
Just to be sure, the "automatic" templateName will create the same ` DataTypes::Name()+ std::string(",")+ImageTypes::Name()`  right ?

Reply:
It is my understanding. If I am motivated, I would like to add unit tests.